crates/aptos-rosetta/src/main.rs

```diff
@@ -3,7 +3,7 @@
 
 #![forbid(unsafe_code)]
 
-use aptos_config::config::ApiConfig;
+use aptos_config::config::{ApiConfig, DEFAULT_MAX_PAGE_SIZE};
```

Comment:
not for this diff, but I am wondering how do we make there being servers with different configs , and clients working with all of them? (and clients continuing to work if we change configs)

should we have some api call (like on index) return this value, so that client can know what to use? should the clients understand too small pagination errors, and adjust that way?

Reply:
Luckily for this one application of Rosetta, it's built to run in tandem with a full node.  So, it doesn't hit this kind of problem.

I think in the future we can make an API to return those kinds of limits.